src/test/java/com/bitmovin/api/examples/CreateMp4AndMovEncoding.java

```diff
@@ -0,0 +1,180 @@
+package com.bitmovin.api.examples;
+
+import com.bitmovin.api.BitmovinApi;
+import com.bitmovin.api.encoding.AclEntry;
+import com.bitmovin.api.encoding.AclPermission;
+import com.bitmovin.api.encoding.EncodingOutput;
+import com.bitmovin.api.encoding.InputStream;
+import com.bitmovin.api.encoding.codecConfigurations.AACAudioConfig;
+import com.bitmovin.api.encoding.codecConfigurations.H264VideoConfiguration;
+import com.bitmovin.api.encoding.codecConfigurations.MJPEGCodecConfiguration;
+import com.bitmovin.api.encoding.codecConfigurations.enums.PixelFormat;
+import com.bitmovin.api.encoding.codecConfigurations.enums.ProfileH264;
+import com.bitmovin.api.encoding.encodings.Encoding;
+import com.bitmovin.api.encoding.encodings.muxing.MP4Muxing;
+import com.bitmovin.api.encoding.encodings.muxing.MuxingStream;
+import com.bitmovin.api.encoding.encodings.muxing.ProgressiveMOVMuxing;
+import com.bitmovin.api.encoding.encodings.streams.Stream;
+import com.bitmovin.api.encoding.enums.CloudRegion;
+import com.bitmovin.api.encoding.enums.StreamSelectionMode;
+import com.bitmovin.api.encoding.inputs.S3Input;
+import com.bitmovin.api.encoding.outputs.S3Output;
+import com.bitmovin.api.encoding.status.Task;
+import com.bitmovin.api.enums.Status;
+import com.bitmovin.api.exceptions.BitmovinApiException;
+import com.bitmovin.api.http.RestException;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Collections;
+import java.util.HashSet;
+
+public class CreateMp4AndMovEncoding
+{
+    private static String API_KEY = "<YOUR BITMOVIN API KEY>";
+
+    private static CloudRegion CLOUD_REGION = CloudRegion.AWS_EU_WEST_1;
+
+    private static String S3_INPUT_ACCESSKEY = "<YOUR S3 INPUT ACCESS KEY>";
+    private static String S3_INPUT_SECRET_KEY = "<YOUR S3 INPUT SECRET KEY>";
+    private static String S3_INPUT_BUCKET_NAME = "<YOUR S3 INPUT BUCKET NAME>";
+    private static String S3_INPUT_PATH = "path/to/your/input/file.mkv";
+
+    private static String S3_OUTPUT_ACCESSKEY = "<YOUR S3 OUTPUT ACCESS KEY>";
+    private static String S3_OUTPUT_SECRET_KEY = "<YOUR S3 OUTPUT SECRET KEY>";
+    private static String S3_OUTPUT_BUCKET_NAME = "<YOUR S3 OUTPUT BUCKET NAME>";
+    private static String OUTPUT_BASE_PATH = "/your/output/base/path/";
+
+    private static String MP4_OUTPUT_FILE_NAME = "myMp4Output.mp4";
+    private static String MOV_OUTPUT_FILE_NAME = "myMovOutput.mov";
+
+    private static BitmovinApi bitmovinApi;
+
+    @Test
+    public void testCreateMp4AndMovEncoding() throws URISyntaxException, BitmovinApiException, UnirestException, IOException, RestException, InterruptedException
```

Comment:
I think it would be good to rename the test to something containing keyframe archive.